integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1392,6 +1393,18 @@ protected void initialize(Map<String, Object> inputDomainMap) throws Exception {
     TestUtils.exec(
         "cp -rf " + BaseTest.getProjectRoot() + "/kubernetes/samples " + BaseTest.getResultDir());
 
+    if (inputDomainMap.containsKey("customDomainTemplate")) {
+      Files.copy(
+          Paths.get((String) inputDomainMap.get("customDomainTemplate")),
+          Paths.get(BaseTest.getResultDir() + "/samples/scripts/common/domain-template.yaml"),
+          StandardCopyOption.REPLACE_EXISTING);
+    }
+    logger.info("Domain Template");
+    byte[] readAllBytes =
+        Files.readAllBytes(
+            Paths.get(BaseTest.getResultDir() + "/samples/scripts/common/domain-template.yaml"));
+    logger.info(new String(readAllBytes, StandardCharsets.UTF_8));
```

Comment:
If this is only for debugging, change the log level to debug after testing as it will be very verbose.

Reply:
done